setup/install_upstream_rust_keylime/test.sh

```diff
@@ -23,7 +26,21 @@ rlJournalStart
             rlRun "sed -i 's%/sys/kernel/security/tpm0/binary_bios_measurements%${TPM_BINARY_MEASUREMENTS}%' src/common.rs"
             rlRun "sed -i 's%/sys/kernel/security/tpm0/binary_bios_measurements%${TPM_BINARY_MEASUREMENTS}%' src/main.rs"
         fi
+
+        if [ -z "$KEYLIME_RUST_CODE_COVERAGE" ]; then
```

Comment:
Hi @ansasaki , could you please take a look at this code block? I am wondering if all added lines are necessary or whether some could possibly duplicate what is already configured in a rust-keylime repository.

Reply:
Unfortunately, this is still necessary as it is not possible to set the measured boot log location via configuration or environment variable. We should probably add an option to allow setting the path.